atoMEC/staticKS.py

```diff
@@ -736,8 +750,9 @@ def calc_S_unbound(orbs):
         -----
         Currently only "ideal" (uniform) treatment of unbound electrons is supported.
 
-        .. math:: S_{ub} = \sum_\sigma \frac{V}{\sqrt{2}\pi^2}
-        where :math: 'I_{1/2}' is the complete Fermi-Dirac integral of order 1/2
+        .. math::
+            S_{ub} = \sum_\sigma \frac{V}{\sqrt{2}\pi^2} I_{1/2}(\mu,\beta)
```

Comment:
Change to
```suggestion
            S_\mathrm{ub} = \sum_\sigma \frac{V}{\sqrt{2}\pi^2} I_{1/2}(\mu,\beta)
```
in order to conform with `S_\mathrm{b}` from line 706.